pkgs/applications/misc/HentaiAtHome/default.nix

```diff
@@ -27,11 +46,12 @@ buildGraalvmNativeImage rec {
     popd
   '';
 
+  strictDeps = true;
+
   meta = with lib; {
     homepage = "https://ehwiki.org/wiki/Hentai@Home";
     description =
       "Hentai@Home is an open-source P2P gallery distribution system which reduces the load on the E-Hentai Galleries";
-    sourceProvenance = with sourceTypes; [ binaryBytecode ];
```

Comment:
Appears clean now by my estimation :+1:

Reply:
Yes, instead of fetching prebuild `.jar` we now build the sources using jdk first.